.github/workflows/check-dependencies.yaml

```diff
@@ -0,0 +1,67 @@
+name: Build and test linkml-runtime package dependencies
+
+on:
+  schedule:
+    - cron: '0 5 * * 1'  # once per week on Monday at 05:00 UTC
+  workflow_dispatch:
+  # Allows you to run this workflow manually from the Actions tab
+    types: trigger-run-check-dependencies
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
Off-topic but there's not need to test 3.7 anymore. It has been EOL [for a little while now](https://devguide.python.org/versions/).

Reply:
removed from two github actions